### PR TITLE
Setting up repo

### PR DESCRIPTION
**Pull request by devngc**
Hi @mostaphaRoudsari, 
Raising this PR to setup the repo. I am guessing there's an automated way to create a docker image of honeybee-vtk.

### CODE REVIEW

File: pollination/honeybee_vtk/__init__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+"""Honeybee Energy plugin for Pollination."""
+from pollination_dsl.common import get_docker_image_from_dependency
+
+# set the version for docker image dynamically based on honeybee-vtk version
+# in dependencies
+image_id = get_docker_image_from_dependency(
+    __package__, 'honeybee-vtk', 'ladybugtools'
```
Comment (mostaphaRoudsari) [inline pollination/honeybee_vtk/__init__.py:7]: Let's open an issue on honeybee-vtk not to forget building and deploying the Docker image

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    maintainer_email='devang@ladybug.tools, info@ladybug.tools',
+    packages=setuptools.find_namespace_packages(
+        include=['pollination.*'], exclude=['tests', '.github']
+    ),
+    install_requires=requirements,
+    use_scm_version=True,
+    setup_requires=['setuptools_scm'],
+    url='https://github.com/pollination/honeybee-vtk',
+    project_urls={
+        'icon': 'https://raw.githubusercontent.com/ladybug-tools/artwork/master/icons_bugs/grasshopper_tabs/Honeybee.png',
```
Comment (mostaphaRoudsari) [inline setup.py:26]: Should we create a new icon for Honeybee VTK?

**Review by mostaphaRoudsari — APPROVED** [review]
Looks good to me. I think we should make sure that we at least have a single command and the Docker image before pushing it to Pollination.

I would remove the ci.yaml for now so it doesn't trigger a release until then.

File: pollination/honeybee_vtk/__init__.py (hunk 1 above)
Comment (devngc) [reply pollination/honeybee_vtk/__init__.py:7]: https://github.com/ladybug-tools/honeybee-vtk/issues/183

File: setup.py (hunk 2 above)
Comment (devngc) [reply setup.py]: I added the vtk icon

Comment (mostaphaRoudsari) [reply setup.py]: This needs to be a url. I see that you are using a local path in the repository. That will not work. You can use this url instead: https://raw.githubusercontent.com/pollination/honeybee-vtk/master/assets/logo/vtk.png

Comment (devngc) [reply setup.py]: I fixed this. Thanks!

**Pull request merged**